src/wasm/thorvgwasm.cpp

```diff
@@ -55,16 +54,10 @@ class __attribute__((visibility("default"))) ThorvgWasm
 
         /* get default size */
         mPicture->viewbox(nullptr, nullptr, &w, &h);
-
-        pw = mDefaultWidth;
-        ph = mDefaultHeight;
         mDefaultWidth = static_cast<uint32_t>(w);
         mDefaultHeight = static_cast<uint32_t>(h);
 
-        if (pw != mDefaultWidth || ph != mDefaultHeight) {
-            updateScale();
-        }
-
+        updateScale();
```

Comment:
looks like updateSize() calls updateScale() internally.
Does it really necessary?

Reply:
updateSize calls updateScale only if canvas size is changed.
So this is necessary.